calibration/calibration.py

```diff
@@ -9,7 +9,7 @@
 
 # Local imports ... should tidy up later
 thisdir = sc.thispath(__file__)
-rabm = sc.importbypath(thisdir.parent / 'rotaABM.py')
+rs = sc.importbypath(thisdir.parent / 'rotasim/rotasim.py')
```

Comment:
now that it's a module you shouldn't need this, just `import rotasim as rs` should work